src/components/Banner/Banner.tsx

```diff
@@ -18,7 +18,7 @@ export const bannerTypesArray = [
   "recommendation",
   "warning",
 ] as const;
-export type BannerTypes = typeof bannerTypesArray[number];
+export type BannerTypes = (typeof bannerTypesArray)[number];
```

Comment:
Out of curiosity, did you do this or a linter?

Reply:
I have save on format enabled, pretty sure the formatting came from either prettier or eslint. I can revert the change though if you prefer.

Reply:
Yeah, can you try to? When it's compiled, it doesn't seem to change anything but for consistency since other types are written without parenthesis.